client/src/components/popup/PopupModal.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './PopupModal.scss';
+import XMark from '../../assets/misc/xmark-solid-white.svg';
+
+const PopupModal = ({
+  trigger,
+  setTrigger,
+  showHeading,
+  heading,
+  children,
+  bgWidth,
+  bgHeight,
+  containerWidth,
+  containerTop,
+}) => {
+  return trigger ? (
+    <div className="popup-background" style={{ width: `${bgWidth}`, height: `${bgHeight}` }}>
+      <div
+        className="popup-container"
+        style={{ width: `${containerWidth}`, top: `${containerTop}` }}
```

Comment:
```
        style={{ width: `${containerWidth}`, top: `${containerTop}` }}
```
hi @jameskokoska quick question! i'm trying to turn the component i made for the password popup, into a "reusable" component, and I want to pass another set of parameters for mobile/tablet display but i don't know the syntax 🤔

Reply:
so container width and container top would vary based on if it's mobile or not?
The best way to do this is to use CSS classes

Reply:
yep, for the forgot password popup, i just wanted to make the width wider in mobile, so it's easier for people to enter emails... and also container top is kinda weird because the height of the login page with 150vh (in desktop) and then 100vh (in mobile), so i wanted to adjust the placement of the popup a little...😅